datadog_checks_downloader/datadog_checks/downloader/download.py

```diff
@@ -332,6 +332,9 @@ def get_wheel_relpath(self, standard_distribution_name, version=None):
 
         # Otherwise, fuhgedaboutit.
         if not href:
-            raise PythonVersionMismatch(standard_distribution_name, version, this_python, python_tags)
+            if ignore_python_version:
+                href = list(python_tags.values())[0]
```

Comment:
So much cleaner now, thx!

Is it safer to index the dictionary here instead?

Reply:
Nope, this is the same thing!